docs/dsp-tools-excel.md

```diff
@@ -74,72 +74,82 @@ For further information about properties, see [here](./dsp-tools-create-ontologi
+![img-list-german-example.png](assets/images/img-list-german-example.png)
 
 Only Excel files with file extension `.xlsx` are considered. All Excel files have to be located in the same directory. 
 When calling the `excel` command, this folder is provided as an argument to the call. The language of the labels has 
-to be provided in the Excel file's file name after an underline and before the file extension, p.ex. `liste_de.xlsx` 
-would be considered a list with German (`de`) labels, `list_en.xlsx` a list with English (`en`) labels. The language 
-has to be a valid ISO 639-1 or ISO 639-2 language code.
+to be provided in the Excel file's file name after an underline and before the file extension, e.g. 
+`Beschreibung_de.xlsx` would be considered a list with German (`de`) labels, `description_en.xlsx` a list with 
+English (`en`) labels. The language has to be one of {de, en, fr, it}.
 
-The following example shows how to create a JSON list from two Excel files which are in a directory called `lists`. 
+The following example shows how to create a JSON list from two Excel files which are in a directory called `listfolder`. 
 The output is written to the file `list.json`.
 
 ```bash
-dsp-tools excel lists list.json
+dsp-tools excel listfolder list.json
 ```
 
-The two Excel files `liste_de.xlsx` and `list_en.xlsx` are located in a folder called `lists`. `liste_de.xlsx` 
-contains German labels for the list, `list_en.xlsx` contains the English labels.
+The two Excel files `Beschreibung_de.xlsx` and `description_en.xlsx` are located in a folder called `lists`.
 
 ```
-lists
-    |__ liste_de.xlsx
-    |__ list_en.xlsx
+listfolder
+    |__ Beschreibung_de.xlsx
+    |__ description_en.xlsx
 ```
 
-For each list node, the `label`s are read from the Excel files. The language code, provided in the file name, is then 
-used for the labels. As node `name`, a simplified version of the English label is taken if English is one of the 
+For each list node, the labels are read from the Excel files. The language code, provided in the file name, is then 
+used for the labels. As node name, a simplified version of the English label is taken if English is one of the 
 available languages. If English is not available, one of the other languages is chosen (which one depends on the 
 representation of the file order). If there are two node names with the same name, an incrementing number is appended to
 the `name`.
 
 ```JSON
 {
-  "name": "sand",
+  "name": "description",
   "labels": {
-    "de": "Sand",
-    "en": "sand"
+    "de": "Beschreibung",
+    "en": "description"
   },
   "nodes": [
     {
-      "name": "fine-sand",
-      "labels": {
-        "de": "Feinsand",
-        "en": "fine sand"
-      }
-    },
-    {
-      "name": "medium-sand",
-      "labels": {
-        "de": "Mittelsand",
-        "en": "medium sand"
-      }
-    },
-    {
-      "name": "coarse-sand",
+      "name": "first-sublist",
```

Comment:
In my view, it is confusing to use the term "sub-list". The node called "first-sublist" here is the actual list. I would suggest using "list" or "my-list" or something similar.

Reply:
As I understand, the contents of Excel column A are already nodes/sublists. The top-level list name is defined by the file name.  
In every case, my test data behaves that way...